src/constants/properties.js

```diff
@@ -18,16 +18,19 @@ export const faithBased = {
   label: 'Faith-based',
 }
 
-export const serviceSite = {
-  path: 'json.contacts.service_site.value.addrtxtlines',
-  label: 'Service site',
-}
-
 export const contacts = {
   path: 'json.contacts',
   label: 'To get connected',
 }
 
 export const hours = {
   path: 'json.hours',
+  label: 'Hours'
 }
+
+export const services = {
+  path: 'json.service_description.service_catagories[0].keys',
```

Comment:
And `service-catagories` always seems to be there. This is something that will change as well when we simplify the structure.